### DISCUSSION TITLE
Tags color not visible in custom fields?

### DISCUSSION THREAD

**Discussion by e95fja** [netbox-community/netbox#16541] [General]
Hi,
I am using Tags to classify IP prefixes. The colors make it very nice.

Instead of using the default Tag column I tried to add a separate column (called CustomField) in the picture below. 
I still want to use Tags with colors. Using Tags works fine but the color is not seen in the table.

In the table below I have chosen the same TEST-TAG but as seen the color is seen in the Tags column but not in 
the CustomField column.

Any ideas if this is possible to achieve?

BR
Fredrik
Netbox version:  v3.7.5

![Netbox_tags_color](https://github.com/netbox-community/netbox/assets/2517978/2284535d-7f13-4aae-8349-d2e89d326274)

**Comment by jeremystretch** [comment]
No, it's not possible; custom fields treat tags like any other object. Though I'm not sure why you would ever want to do this, since you can already apply tags directly to the object.

**Comment by e95fja** [comment]
Hi,
Thanks for the answer!

The reason is that it would be nice to have several columns in that table all using Tags.

Maybe one for ConfidentialityClasification, one for SecurityClassifcation etc. 

It is possible already today but without the nice colors of the Tags. 

As in the picture above the red color stands out in the Tags column. But not in the CustomField column.
